extension/controller/translation/Translation.js

```diff
@@ -9,6 +9,10 @@
 class Translation {
     constructor (mediator){
         this.translationsMessagesCounter = 0;
+        this.TRANSLATION_INTERVAL = 100; // ms
+        this.MAX_TRANSLATION_MSGS = 500; // max translations to process per batch
```

Comment:
how did you pick this number? maybe we should play with it and see what's optimal, it might also depend on the user's hardware

Reply:
just randomly experimenting, if we could find an equation here based on the platform factors would be the ideal if we have the tools available